doc/modules/cross_validation.rst

```diff
@@ -554,3 +554,20 @@ Cross validation and model selection
 Cross validation iterators can also be used to directly perform model
 selection using Grid Search for the optimal hyperparameters of the
 model. This is the topic of the next section: :ref:`grid_search`.
+
+Cross validation of time series data
+====================================
+
+Time series data is known for the correlation between observations, so 
```

Comment:
"known for" -> "characterised by"

"between observations that are near in time (*autocorrelation*)